option.go

```diff
@@ -196,6 +197,12 @@ func Runner(name, dsn string, opts ...httpRunnerOption) Option {
 			r.client.CheckRedirect = notFollowRedirectFn
 		}
 		r.multipartBoundary = c.MultipartBoundary
+		if c.Timeout != "" {
+			r.client.Timeout, err = duration.Parse(c.Timeout)
+			if err != nil {
+				return fmt.Errorf("timeout in HttpRunnerConfig is invalid: %w", err)
+			}
+		}
```

Comment:
What about putting in a timeout setting before calling the newHttpValidator function?
It seems like unnecessary logic or duplicate code, what do you think?

Reply:
> It seems like unnecessary logic or duplicate code, what do you think?

I think the duplication of code is an issue with the current runn structure, so there is no way around it.